madgiraffestudios/src/components/Gallery/Gallery.js

```diff
@@ -0,0 +1,105 @@
+      thumbnail: 'https://firebasestorage.googleapis.com/v0/b/madgiraffestudiossite.appspot.com/o/nai_shoot.jpg?alt=media&token=5feb0e35-321c-4948-8f69-0f99f4e87b25',
+      startIndex: 0,
+      description: 'Nothing is ever black, white or gray. Let your colors shine! '
+    },
+    {
+      original: 'https://firebasestorage.googleapis.com/v0/b/madgiraffestudiossite.appspot.com/o/couple.jpg?alt=media&token=26bb85a4-956c-465e-a17b-06fb91120650',
+      thumbnail: 'https://firebasestorage.googleapis.com/v0/b/madgiraffestudiossite.appspot.com/o/couple.jpg?alt=media&token=26bb85a4-956c-465e-a17b-06fb91120650',
+      startIndex: 1,
+      description: 'Strong Black Love'
+      
+    },
+    {
+      original: 'https://firebasestorage.googleapis.com/v0/b/madgiraffestudiossite.appspot.com/o/beach_shoot2.jpg?alt=media&token=0a46be1f-98ce-4cd9-b878-16aded6a43b8',
+      thumbnail: 'https://firebasestorage.googleapis.com/v0/b/madgiraffestudiossite.appspot.com/o/beach_shoot2.jpg?alt=media&token=0a46be1f-98ce-4cd9-b878-16aded6a43b8',
+      startIndex: 2,
+      description: 'A melanin queen '
+      
+    },
+    {
+      original: 'https://firebasestorage.googleapis.com/v0/b/madgiraffestudiossite.appspot.com/o/beach_shoot2.jpg?alt=media&token=0a46be1f-98ce-4cd9-b878-16aded6a43b8',
+      thumbnail: 'https://firebasestorage.googleapis.com/v0/b/madgiraffestudiossite.appspot.com/o/beach_shoot2.jpg?alt=media&token=0a46be1f-98ce-4cd9-b878-16aded6a43b8',
+      startIndex: 2,
+      description: 'A melanin queen '
+      
+    },
+    {
+      original: 'https://firebasestorage.googleapis.com/v0/b/madgiraffestudiossite.appspot.com/o/beach_shoot2.jpg?alt=media&token=0a46be1f-98ce-4cd9-b878-16aded6a43b8',
+      thumbnail: 'https://firebasestorage.googleapis.com/v0/b/madgiraffestudiossite.appspot.com/o/beach_shoot2.jpg?alt=media&token=0a46be1f-98ce-4cd9-b878-16aded6a43b8',
+      startIndex: 2,
+      description: 'A melanin queen '
+      
+    },
+    {
+      original: 'https://firebasestorage.googleapis.com/v0/b/madgiraffestudiossite.appspot.com/o/beach_shoot2.jpg?alt=media&token=0a46be1f-98ce-4cd9-b878-16aded6a43b8',
+      thumbnail: 'https://firebasestorage.googleapis.com/v0/b/madgiraffestudiossite.appspot.com/o/beach_shoot2.jpg?alt=media&token=0a46be1f-98ce-4cd9-b878-16aded6a43b8',
+      startIndex: 2,
+      description: 'A melanin queen '
+      
+    },
+    
+    
+
+  ];
+
+  const galleryImages = gallery.map((img) => 
+    <div className="col-md-4 mb-4 image-container img-wrap" key={img.startIndex} onClick={() =>imageClick(img.startIndex)}>
+      <img src={img.original} alt={img.description} width="310px" height="380px" className="rounded" />
+    <div class="after">
+      <span>{img.description}</span>
+    </div>
+  </div>
+
+  );
+
+
+  const [modal, setModal] = useState(false);
+  const [imageIndex, setImgIndex] = useState();
+  const toggle = () => setModal(!modal);
```

Comment:
- [ ] This function can be removed since it's not being called anymore

Reply:
Resolved